src/styles/App.scss

```diff
@@ -162,6 +156,16 @@ a {
       margin: 10px 0;
       background-color: $warning-fill;
     }
+    .calculator-params {
+      label {
+        @include media-breakpoint-up(lg) {
+          min-height: 3em;
```

Comment:
This doesn't work at all browser sizes:
![image](https://user-images.githubusercontent.com/5877421/95690272-4b07c880-0bcb-11eb-9a3a-0c20f1bd6e38.png)

Ok for now, I can see if I can figure out something with tables to make this more robust.

Reply:
Hm. Why tables? I thought the general thinking is table shouldn't be used for layout. That's what flexbox is for these days, I thought. Curious your thinking there.

Reply:
I meant the <Row> and <Col> tags that come from bootstrap, not a literal table. But from the slack it sounds like the direction is a single column anyway, so this is moot.

Reply:
Yeah lets head toward 1 column. I'll do that in a separate PR though.